osquery/tables/system/posix/smbios_utils.cpp

```diff
@@ -142,5 +148,21 @@ std::string dmiString(uint8_t* data, uint8_t* address, size_t offset) {
 
   return std::string(bp);
 }
+
+std::string dmiBitFieldToStr(size_t bitField,
+                             const std::map<uint8_t, std::string>& table) {
+  std::stringstream ss;
```

Comment:
Does this need a `stringstream`?

Reply:
No I don't.  I think I see your point.  Since all i'm doing is concatenating the string, I'll just use `std::string` directly :+1: